src/mina-signer/tests/rosetta.test.ts

```diff
@@ -23,6 +25,25 @@ describe('Rosetta', () => {
     client = new Client({ network: 'mainnet' });
   });
 
+  it('generates a valid signature from a hex string', () => {
+    let { r, s } = signatureFromHex(rosettaTnxMockSignature);
+    expect(r).not.toBeNaN();
+    expect(r).not.toBeFalsy();
+    expect(s).not.toBeNaN();
+    expect(s).not.toBeFalsy();
+
+    // TODO we could go further and use validate method on the return?
+    // or leave as is with the assumption that the above hex is known to be valid
+  });
+
+  it('generates a valid hex string from a signature', () => {
+    let signature = signatureFromHex(rosettaTnxMockSignature);
+    let hex = signatureToHex(signature);
+
+    // fails
+    expect(hex).toEqual(rosettaTnxMockSignature);
```

Comment:
as stated this fails with a single char being different than what is expected (see test output). @mitschabaude interesting... that or my expectation is incorrect 🤷‍♂️

Reply:
Interesting, that test should definitely pass